DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/Acquirente/GestioneAccountAcquirente.tex

```diff
@@ -80,29 +149,31 @@
             \item Si posiziona nel campo di inserimento vuoto della conferma nuova password;
             \item Inserisce la stessa password che è stata inserita nel campo conferma password.
         \end{itemize}
-    \item \textbf{Scenario alternativo:} l'acquirente non inserisce alcuna password e lascia il campo di inserimento conferma nuova password vuoto. In questo caso non verrà modificata la password.
-    \item \textbf{Estensioni:} \begin{enumerate}[label=\lett] 
-    \item l'acquirente inserisce una password di conferma diversa da quella inserita nel campo di inserimento nuova password. In questo caso:
-    \begin{itemize}
-        \item (UC) - Verrà visualizzato il messaggio di errore di password e password di conferma diverse;
-        \item Verrà impedita la modifica delle informazioni.
-    \end{itemize}
-\end{enumerate}
+    \item \textbf{Scenari alternativi:} 
+    \begin{enumerate}
+        \item L'acquirente non inserisce alcuna password e lascia il campo di inserimento conferma nuova password vuoto. In questo caso non verrà modificata la password.
+    \end{enumerate}
+    \item \textbf{Estensioni:} 
+    \begin{enumerate}[label=\lett] 
+        \item l'acquirente inserisce una password di conferma diversa da quella inserita nel campo di inserimento nuova password. In questo caso:
+        \begin{itemize}
+            \item (UC) - Verrà visualizzato il messaggio di errore di password e password di conferma diverse;
+            \item Verrà impedita la modifica delle informazioni.
+        \end{itemize}
+    \end{enumerate}
```

Comment:
questa estensione secondo me è come il discorso delle date in filtraggio, è il padre che se che ha il figlio "nuova password" e che lo controlla con "conferma nuova password" che dici?

Reply:
Va bene, hai ragione